sites/svelte-5-preview/src/lib/Repl.svelte

```diff
@@ -77,7 +77,7 @@
 	/** @type {import('svelte/compiler').CompileOptions} */
 	const DEFAULT_COMPILE_OPTIONS = {
 		generate: 'client',
-		dev: false
+		dev: true
```

Comment:
reverting this. the JS output tab should show prod output by default

```suggestion
		dev: false
```